Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.cpp

```diff
@@ -39,7 +43,14 @@ namespace VehicleDynamics
                         AZ::Edit::UIHandlers::Default,
                         &SimplifiedDriveModel::m_speedPid,
                         "Speed PID",
-                        "Configuration of speed PID controller");
+                        "Configuration of speed PID controller")
+                    ->DataElement(AZ::Edit::UIHandlers::Default, 
+                        &SimplifiedDriveModel::maxSpeedImpulse, 
+                        "Max speed impulse", "Speed torque impulse limit [0, INF]. Set to 0.0 to disable.")
```

Comment:
Use attribute Min here (input will be limited). See VehicleLimits.

Reply:
Maybe "Maximum torque force that can be applied to wheels regardless of controller output"